_data/organizers.yml

```
@@ -21,3 +21,12 @@
   link: "dcwalker.ca"
   twitter: dcwalk_
   github: dcwalk  
+
+- organizerID: mauve
+  name: "Mauve"
+  bio: >-
+    is a tech entusiast with a passion for decentralization. They use the web and peer to peer protocols like Dat to create local first software to give people more control over their data and to make them less dependant on centralized services. They are active in open source organizing and enjoy reaching out to communities and bridging between them. Their passion project is Local First Cyberspace - a peer to peer augmented reality platform that uses the web.
```

Just change to punctuation to match bio above!

```suggestion
    is a tech entusiast with a passion for decentralization. They use the web and peer to peer protocols like Dat to create local first software to give people more control over their data and to make them less dependant on centralized services. They are active in open source organizing and enjoy reaching out to communities and bridging between them. Their passion project is Local First Cyberspace&mdash;a peer to peer augmented reality platform that uses the web.
```